nlmod/gwt/gwt.py

```diff
@@ -0,0 +1,422 @@
+import logging
+import numbers
+
+import flopy
+import xarray as xr
+
+from ..dims import grid
+from ..gwf.gwf import _dis, _disv, _set_record
+
+logger = logging.getLogger(__name__)
+
+
+def _get_var_from_ds_attr(ds, varname, attr=None, var=None, warn=True):
+    """Internal function to parse variable from dataset attributes.
+
+    Parameters
+    ----------
+    ds : xarray.Dataset
+        dataset containing model data
+    varname : str
+        name of the variable
```

Comment:
name of the variable in a flopy package?

Reply:
:+1: